src/provider.ts

```diff
@@ -24,6 +24,12 @@ export interface RequestInvoiceArgs {
   defaultMemo?: string;
 }
 
+export interface KeysendArgs {
+  destination: string;
+  customRecords: Record<string, string>;
+  amount: string;
```

Comment:
For consistency with other `webln` methods dealing with amounts, we should accept `string | number`
```suggestion
  amount: string | number;
```